pandas/core/tools/timedeltas.py

```diff
@@ -90,7 +88,6 @@ def to_timedelta(
     | Series,
     unit: UnitChoices | None = None,
     errors: DateTimeErrorChoices = "raise",
-    # returning Any for errors="ignore"
 ) -> Timedelta | TimedeltaIndex | Series | NaTType | Any:
```

Comment:
Can the `Any` be removed then?

Reply:
No I tried that and all kinds of typing checks started failing